ftplugin/supercollider.vim

```diff
@@ -200,9 +200,6 @@ endfunction
 let s:sclangStarted = 0
 
 function SClangStart()
```

Comment:
This can go: This hotfix only protected from part of the the process buildup problems that are fixed at the root cause above.